GliaWidgets/Sources/RemoteConfiguration/RemoteConfiguration+Chat.swift

```diff
@@ -23,6 +23,7 @@ extension RemoteConfiguration {
 
     struct Gva: Codable {
         let persistentButton: GvaPersistentButton?
+        let quickReplyButton: Button?
```

Comment:
I appreciate that you have called it quick reply and not something like qrButton :D